packages/teleport-project-generator/src/file-handlers.ts

```diff
@@ -182,6 +191,31 @@ const createHTMLEntryFileChunks = (uidl: ProjectUIDL, options: EntryFileOptions)
     }
   })
 
+  // Stencil need to inject and point out the generated build files
```

Comment:
this functionality can be used by any project generator, I think it's ok to have it as generic as it is, without explaining the specific need for stencil, so you can change into: `For frameworks that need to ....`